### PR TITLE
Refactor content to be generated from data files

### PR DESCRIPTION
**Pull request by AndreasMadsen**
What needs to happen after this is merged:

* Protect the `master` branch
* Give everybody right to create a `PR`
* Add https://github.com/marketplace/actions/yaml-schema-validator to our GitHub actions. -- The schemas are already implemented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,32 +2,55 @@
...
 
-## Install
+## Maintenance
 
-```bash
-bundle install
-```
+If there are updates to people, publications, or news you can in most cases easily update this information yourself.
+If you need help you can containt the current maintainer. If you don't speak french and need to provide french information,
+just create aa PR with the english content and contact the french maintainer to help you.
```
Comment (saujasv) [inline README.md:9]: containt -> contact
french -> French
aa -> a

File: _includes/collage.html [hunk 2]
```diff
@@ -1,142 +1,19 @@
...
-
-
-        </div><!-- /end row -->
-    </div><!-- /end container -->
-</section>
\ No newline at end of file
+        <div class="team-members">
+            {% for person in site.data.people %}
+            <div class="member-details"
+                {% if person.collage_column_start != nil %}
```
Comment (saujasv) [inline _includes/collage.html:12]: It might help to document the logic behind `collage_column_start` somewhere.

**Review by saujasv — APPROVED** [review]

File: _includes/collage.html (hunk 2 above)
Comment (AndreasMadsen) [reply _includes/collage.html:12]: It is already documented here:
https://github.com/chandar-lab/chandar-lab.github.io/pull/4/files#diff-81afb82f1179e89eb574f60daa8bed2d30f36d54bdc9672d5b308c4885acc3f3R10-R13

File: _data/people.yaml [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+# The picture, website, cosupervisor, and topics fields are optional.
+# The topics field must contain both english (en) and french (fr) translations.
+# The title field must be either: pi, postdoc, phd, or intern
+# Note that `intern` will be placed under "Interns/RAs" so this includes Research Assistants too.
+
+# The order in this file is the order of people in the collage
+# Note that to add blank spaces in the collage the collage_column_start property can be set.
+# collage_column_start indicates at which column the person should appear, sucessing people
+# will the follow after possibly spaces blank.
```
Comment (saujasv) [inline _data/people.yaml:13]: sucessing people will follow after possibly blank spaces.

File: _includes/collage.html (hunk 2 above)
Comment (saujasv) [reply _includes/collage.html:12]: I have suggested some elaboration on that. If it is helpful, you could include it.

File: _data/people.yaml (hunk 3 above)
Comment (AndreasMadsen) [reply _data/people.yaml:13]: Hmm,`collage_column_start` does not necessarily indicate a new row. You could have a gap in the middle by setting `collage_column_start: 2` and `collage_column_start: 4` on two succeeding items.

File: README.md (hunk 1 above)
Comment (AndreasMadsen) [reply README.md]: Thanks :)

File: README.md [hunk 4]
```diff
@@ -2,32 +2,55 @@
...
+1. Add them to [`_data/people.yml`](https://github.com/chandar-lab/chandar-lab.github.io/edit/master/_data/people.yml)
+2. Add their profile picture too [`assets/images/`](assets/images/). Remember to upload in a reasonable small resolution, to keep the bandwidth low.
+2. Create a pull request.
 
-When a publication is added or changed, update [`publications.md`](publications.md). Remember to update the `{: reversed="reversed" start="[number]"}` instruction.
+If someone leaves the group, then:
 
-### News
+1. Remove them from [`_data/people.yml`](https://github.com/chandar-lab/chandar-lab.github.io/edit/master/_data/people.yml)
+2. Remove their profile picture from [`assets/images/`](assets/images/).
```
Comment (olewicki) [inline README.md:38]: and remove from collage (it seems to be automatic now, is it?)

Comment (AndreasMadsen) [reply README.md:38]: Yes, the collage is automatically generated.

File: _data/people.yaml (hunk 3 above)
Comment (saujasv) [reply _data/people.yaml:13]: Right, I think you could just mention this in the doc and should be good for now.

Comment (AndreasMadsen) [reply _data/people.yaml]: I've extended the documentation significantly. With several examples.

**Pull request merged**